scripts/ci-boot-simulator.sh

```diff
@@ -48,3 +48,9 @@ esac
 
 echo "Booting simulator $SIMULATOR"
 xcrun simctl boot "$SIMULATOR"
+
+# Wait for the simulator to boot
+xcrun simctl bootstatus "$SIMULATOR"
```

Comment:
Maybe the issue was in waiting for the simulator boot, which was removed when removing the sim creation; only time will tell.